packages/jsii-dotnet-jsonmodel/NuGet.Metadata.props.t.js

```diff
@@ -0,0 +1,17 @@
+const package = require('./package.json');
+
+process.stdout.write(`<Project>
+  <PropertyGroup>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <PackageOutputPath>..\\..\\bin\\$(Configuration)\\NuGet\\</PackageOutputPath>
```

Comment:
Will the backslash be a problem on UNIX systems?

Reply:
Nope, tested on WSL and `superchain`.